### ISSUE TITLE
Reproduce the performance of PSPNet on segmentation benchmark

### ISSUE THREAD

**Issue by baibaidj** [catalyst-team/catalyst#849]
## ❓ Questions and Help
I want to use the catalyst framework to reproduce the PSPNet on the cityscapes dataset. With the [official repo of PSPNet](https://github.com/hszhao/semseg), I can achieve mIoU of 0.756, very close that reported by the authors of PSPnet. However, I can only get 0.45 mIoU using the catalyast framework. I've checked the input, the preprocessing with albumentations, loss functions and metrics, trying to mimics the official repo as much as possible. 
I was wondering if the catalyst team or other developers have tried to reproduce the the results of major CNNs on semantic segmentation benchmarks like cityscapes, PASCOL VOC or ADE20K. 

I'm still investigating what's going wrong.  

Any insights are welcomed. 

### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
How can I reproduce the performance of PSPNet on cityscapes dataset. 

#### Code
```

dataset=cityscapes

for ARCH in smp.PSPNet; do #smp.Unet smp.FPN
      for encoder in resnet50; do
      log_name=${ARCH}-${encoder}
      batchsize=8
      numworkers=8
      LOGDIR=exp/catalyst-${dataset}/${log_name}/
      mkdir -p ${LOGDIR} 
      USE_WANDB=0 catalyst-dl run \
            --configs ./catalyst_${dataset}/config_${dataset}.yml \
            ./catalyst_${dataset}/transforms.yml \
            --expdir=./catalyst_${dataset} \
            --logdir=$LOGDIR \
            --model_params/model=$ARCH:str \
            --model_params/encoder_name=$encoder:str \
            --monitoring_params/name=${log_name}:str \
            --stages/data_params/batch_size=${batchsize}:int\
            --stages/data_params/num_workers=${numworkers}:int\
            --verbose --distributed --autoresume best\
      done
done

```

#### What have you tried?
I've checked the input, the preprocessing with albumentations, loss functions and metrics, trying to mimics the official repo as much as possible. 

#### What's your environment?
Catalyst version: 20.05
PyTorch version: 1.4.0
Is debug build: No
CUDA used to build PyTorch: 10.1
TensorFlow version: N/A
TensorBoard version: 2.2.1

OS: linux
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
CMake version: version 3.10.2

Python version: 3.7
Is CUDA available: Yes
CUDA runtime version: 10.1.243
Nvidia driver version: 418.56

segmentation-models-pytorch==0.1.0
```


### Additional context
<!-- Add any other context about the problem here. -->

**Comment by Scitator** [comment]
🤔  Looks interesting, don't you mind to join our slack to discuss it more precisely?
https://join.slack.com/t/catalyst-team-core/shared_invite/zt-d9miirnn-z86oKDzFMKlMG4fgFdZafw
